src/actionTypes/article.js

```diff
@@ -0,0 +1,3 @@
+export const CREATE_ARTICLE_SUCCESS = "CREATE_ARTICLE_SUCCESS";
```

Comment:
You have 3 identical action types in ```src/actionTypes/article.js``` and  ```src/actionTypes/auth.js```
Any reason for this?

Reply:
That was an oversight. I'll fix it now.